src/build/brick_proto.rs

```diff
@@ -89,7 +94,7 @@ impl Brick {
                 let alpha_joints = bot.map(|point| fabric.create_joint(point));
                 let omega_joints = top.map(|point| fabric.create_joint(point));
                 for (&alpha_index, &omega_index) in alpha_joints.iter().zip(omega_joints.iter()) {
-                    fabric.create_interval(alpha_index, omega_index, Link::push(ROOT6 * pretenst_factor));
+                    fabric.create_interval(alpha_index, omega_index, Link::push(3.205));
```

Comment:
?

Reply:
Change the number and run the prototype brick operation